enhancements/shared-and-guaranteed-cpu-pools.md

```diff
@@ -88,8 +88,10 @@ kernel tuning applied to each pool.
 guaranteed CPUs.
 * The selection of shared vs. guaranteed CPUs should be transparent to the user - no changes to the pod
 spec should be required.
-* The configuration should be optional and activated with a feature gate.
-* Although the initial usecase for this enhancement is single-node deployments, it should also be usable
+* The feature should be implemented behind a feature gate to ensure no impact to existing functionality.
```

Comment:
Let's move this to a seprate Feature Gate section and also propose a name for it.

Reply:
nvm, I see the section below.